src/v1/accessor.rs

```diff
@@ -6,7 +6,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-enum_number!(
+enum_number! {
     ComponentType {
```

Comment:
This could be more 'rustic' with the following:
```
ComponentType {
    I8 = 5120,
    U8 = 5121,
    F32 = 5126,
    F64 = 5127,
}
```
This would need to be discussed further because it conflicts with `Material` (`FloatVec3` etc). If desired we'll make the change in a future release.